apps/test/util/testUtils.js

```diff
@@ -161,6 +161,10 @@ exports.debugMode = function () {
  * @param {number} top Vertical offset from top left of visualization to drop at
  */
 exports.dragToVisualization = function (type, left, top) {
+  // this delta of 40 is a bit of a mystery, but I think it has to do with our
+  // tests not having CSS applied to them.
+  left = left + 40;
```

Comment:
Turns out this comes from the fact that when we start our drag, we have the dragged item be centered on our cursor. As a result, the drop target ends up being where we drop the center of our dragged item. Our buttons are 80px wide, so this accounts for the element by 40px off from where we expect. Best fix would be to programmatically derive 40 instead of hard coding, so that it works for other element types as well.